docs/Model/ModifyServerRequest.md

```diff
@@ -0,0 +1,10 @@
+# ModifyServerRequest
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**server** | [**\Upcloud\ApiClient\Model\Server**](Server.md) |  | [optional] 
+
+[[Back to Model list]](../README.md#documentation-for-models) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to README]](../README.md)
```

Comment:
All the README links are broken and should be fixed. (location is another directory lover so `../../README.md`)